### DISCUSSION TITLE
The Japanese language OSPO Local Meetup quarterly report #3 (2023/07-09)

### DISCUSSION THREAD

**Discussion by kuwatm** [todogroup/ospology#382] [OSPO Stories]
OSPO Local Meetups in Japanese are held twice a month in Japan.
This quarter, a hybrid meeting was held with In Person in Shin-Yokohama on 14th July. The number of participants ranged from 6 to 16 on some occasions, partly because the schedule changed unusually. On 6th October, the meeting was held on the same day as the OpenChain Japan WG plenary meeting, and over 20 people gathered, so we are hopeful that the number will expand in the next quarter.

 << Strategy & Hosting >> Leader Motai-san
(20230714 #11)(20230728 #13)(20230908 #15)
When discussing OSPO, even when using the same language, the background perception is different and the story may not fit. One reason for this problem is the difference in the relationship between OSS and business. Therefore, from the first order (providing value by providing OSS) to the third order (using OSS as a tool without giving it to users), we discuss examples of products and services, relationships with the OSS community, and actions that OSPO should take toward the community. The discussion has been challenging. Discussion will continue.
Map currently being created:
https://github.com/OpenChain-Project/OpenChain-JWG/blob/master/OSPO/OSPOLocalMeetup-Memo/QuarterlyReport/202309/OSPO%20Japan%20Local%20Meetup%2C%20OSS%20Strategy%2C%2020230930.xlsx

<< OSPO Launch & FAQ >> Leader Owada-san
(20230721 #12)(20230825 #14)(20230922 #16)
 We discussed what we could get from open source activity on each stage of OSPO's two types of Stages, and added acquisitions to the Stages in speech bubbles and completed once. Next, we disscussed about answers for a question "What is OSPO?", and the answers were mapped to OSPO's Stage, and the discussion deepened with the recognition that the person's appearance differs depending on the stage in which he or she is standing. The discussion continues.
Slide currently being created:
https://github.com/OpenChain-Project/OpenChain-JWG/blob/master/OSPO/OSPOLocalMeetup-Memo/QuarterlyReport/202309/OSPO_SimpleQA_20230930.pptx

The materials are also arranged below in Japanese and English for your perusal.
[OpenChain-JWG/OSPO/OSPOLocalMeetup-Memo at master · OpenChain-Project/OpenChain-JWG (](https://github.com/OpenChain-Project/OpenChain-JWG/tree/master/OSPO/OSPOLocalMeetup-Memo)[github.com](http://github.com/)[)](https://github.com/OpenChain-Project/OpenChain-JWG/tree/master/OSPO/OSPOLocalMeetup-Memo)

**Comment by anajsana** [comment]
Thank you @kuwatm !
